examples/install_in_container.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e7b3ff58-6ab9-436a-a457-1bc12d29b7a8",
+   "metadata": {},
+   "source": [
+    "# Containerized Deployment with Docker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aaf3f32f-8f79-4969-ba96-37c7b9c55241",
+   "metadata": {},
+   "source": [
+    "Running NVIDIA FLARE in a Docker container is sometimes a convenient way to ensure a uniform OS and software environment across client and server systems. This can be used as an alternative to the bare-metal Python virtual environment described above and will use a similar installation to simplify transitioning between a bare metal and containerized environment.\n",
+    "\n",
+    "To get started with a containerized deployment, you will first need to install a supported container runtime and the NVIDIA Container Toolkit to enable support for GPUs. System requirements and instructions for this can be found in the NVIDIA Container Toolkit Install Guide <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html>.\n",
+    "\n",
+    "A simple Dockerfile is used to capture the base requirements and dependencies. In this case, we’re building an environment that will support PyTorch-based workflows, in particular the Hello PyTorch with Tensorboard Streaming example. The base for this build is the NGC PyTorch container. On this base image, we will install the necessary dependencies and clone the NVIDIA FLARE GitHub source code into the root workspace directory. To create a Dockerfile, create a file named Dockerfile using any text editor and include the following:\n",
+    "\n",
+    "```\n",
+    "ARG PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:23.02-py3\n",
+    "FROM ${PYTORCH_IMAGE}\n",
+    "\n",
+    "RUN python3 -m pip install -U pip\n",
+    "RUN python3 -m pip install -U setuptools\n",
+    "RUN python3 -m pip install torch torchvision tensorboard nvflare\n",
```

Comment:
no need to install `torch torchvision tensorboard` as the container already has those.